docs/services/productcatalogservice.md

```diff
@@ -0,0 +1,115 @@
+# productcatalog service
+
+This service is responsible to return information about products. The service
+can be used to get all products, search for specific products, or return details
+about any single product.
+
+[Productcatalog service source](../../src/productcatalogservice/)
+
+## Traces
+
+### Initialize trace provider
+
+The OpenTelemetry SDK is initialized from `main` using the `initTracerProvider`
```

Comment:
initTracerProvider -> InitTraceProvider

Reply:
Good catch on this one.  Technically the code should be changed to use lowercase since this function isn't exported.  I'll update accordingly.